src/syscall/js/js.go

```diff
@@ -209,6 +209,7 @@ func ValueOf(x interface{}) Value {
 	}
 }
 
+//go:noescape
```

Comment:
Justification: this will affect whether strings with dynamic contents but statically known length to not escape anymore.